recipes/csa/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+    name: csa
+    version: 1.22.0
+
+source:
+    git_url: https://github.com/sakov/csa-c.git
+    git_tag: v1.22.0
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+    run:
+        - libgcc
```

Comment:
Ack, yes, I am familiar with `libgomp` fun. Let me think about it a little more.